src/api/__specs__/campaigns-spec.js

```diff
@@ -5,12 +5,21 @@ import * as getJSONP from '../../lib/getJSONP';
 describe('campaigns', () => {
   const callback = sinon.spy();
 
-  beforeEach(() => {
+  before(() => {
     sinon.stub(getJSONP, 'default');
   });
 
-  afterEach(() => {
+  after(() => {
     getJSONP.default.restore();
+  })
+
+  afterEach(() => {
+    try{
+      getJSONP.default.reset();
```

Comment:
I don't want to restore it to the previous state as we're still working with it. I do, however, want to reset the test condition. This block flickers often, hence the catch, but having a single spy that resets flickers a whole lot less